### DISCUSSION TITLE
Does JReleaser supports publishing to Maven Central with the new "Publish Via the Central Portal"?

### DISCUSSION THREAD

**Discussion by FDelporte** [jreleaser/jreleaser#1700] [General]
For a new groupid, I created a new account on https://central.sonatype.org following this approach: https://central.sonatype.org/register/central-portal/. Does JReleases already support publishing of Java libraries to Maven central for this new approach? Is an example available, to understand which settings are different?

**Comment by aalmiray** [comment]
Yes, it does since `v1.12.0`. Have a look at https://jreleaser.org/guide/latest/reference/deploy/maven/maven-central.html and https://jreleaser.org/guide/latest/examples/maven/maven-central.html

**Comment by FDelporte** [comment]
Let me rewrite my question: "ofcourse JReleaser supports the new Maven flow, but I didn't immediately find it in the documentation" ;-)

Thanks, and maybe this can be useful for others: https://github.com/jreleaser/jreleaser.github.io/pull/77
